lambdas/indexer/app.py

```diff
@@ -33,6 +34,8 @@
 
 app = AzulChaliceApp(app_name=config.indexer_name)
 
+storage_service = StorageService()
```

Comment:
While this probably makes sense in the long run, it is inconsistent with where this service is currently instantiated, which is within the individual methods that use the service. I think we should stick with this pattern for now. Sorry for not noticing this earlier.